OxfordBodleian/BodNewMSS/BDLaethf35.xml

```diff
@@ -34,9 +37,97 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                ወዝንቱ፡ ረድእ፡ ዘኮነ፡ ሰማዕተ፡ በእንተዝ፡ ወዘሂ፡ ጸሐፈ፡ ዘንተ፡ ንሕነ፡ ነአምር፡ ከመ፡ ጽድቅ፡ ውእቱ፡ ስምዑ። ወዘንተ፡ ገብረ፡ እግዚእ፡
+                                <hi rend="rubric">ኢየሱስ</hi>፡ ወቦ፡ ባዕድኒ፡ ብዙኅ፡ ዘገብረ፡ እግዚእ፡ <hi rend="rubric">ኢየሱስ፡</hi> ወሶበ፡ 
+                                ተ<cb n="b"/>ጽሕፈ፡ በበ፩ዱ፡ እምኢያገመሮ፡ ዓለም፡ ጥቀ፡ እመጻሕፍቲሁ፡ ዝተጽሕፈ፡ 
+                                አሜን<hi rend="rubric" rendition="#partialRubric">፨</hi> <hi rend="rubric" rendition="#partialRubric">፨</hi> <hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                <hi rend="rubric">መልዓ፡ ስብከተ፡ ዮሐንስ፡ ሐዋር</hi>ያ፡ ወልደ፡ ዘብዴዎስ፡ ፩ እም፲ወ፪ቱ፡ ሐዋርያት። ወጸሐፎ፡ በዮናኒ፡
+                                ለሰብአ፡ ሀገረ፡ ኤፌሶን። እምድኅረ፡ ዕርገቱ፡ ለእግዚእነ፡ <hi rend="rubric">ኢየሱስ፡ ክርስቶስ፡</hi>
+                                በሥ<pb n="79v"/><cb n="a"/>ጋ፡ ውስተ፡ ሰማያት። በ፴ ዓመት፡ ወ፰ ዓመተ፡ መንግሥቱ፡ ለኔሮን፡ ቄሣር፡ 
+                                ንጉሠ<supplied reason="omitted">፡</supplied>
+                                <del rend="overUnderlined">ማ</del>ሮሜ፡ ወስብሐት፡ ለእ<hi rend="ligature">ግዚ</hi>አብሔር፡
+                                ወምሕረቱ፡ ይኩን፡ ምስለ፡ ገብሩ፡ <persName ref="PRS14002WaldaS">ወልደ፡
+                                <hi rend="rubric">ሥላሴ፡</hi></persName> ወምስለ፡ ጸሐፊሁ፡ 
+                                <persName ref="PRS14003FereS">ፍሬ፡ <hi rend="rubric">ሰንበት፡</hi></persName> ለዓለመ፡ ዓለም፡ አሜን።
+                            </explicit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="81ra"/>
+                            <title ref="LIT1099Anapho"/>
+                            <textLang mainLang="gez"/>
+                            <!--end of text not photographed-->
+                            <incipit xml:lang="gez">
+                                <locus target="#81ra"/>
+                                <hi rend="rubric">አኰቴተ፡ ቍርባን፡ ዘእግዝእ</hi>ትነ፡ <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ 
+                                አም<hi rend="rubric">ላክ፡ ጽ</hi>ርደት፡
+                                <hi rend="rubric">ዘነበበ፡ በመንፈ</hi>ስ፡ ቅዱስ፡ አባ፡ ህርያቆስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ብህንሳ፡ ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ 
+                                 ምስለ፡ <hi rend="rubric">ገብሩ፡
+                                    <persName ref="PRS14002WaldaS">ወልደ፡ ሥላሴ፡</persName> ለዓለመ<supplied reason="omitted">፡</supplied></hi> 
+                                ዓለም፡ አሜን፡ 
+                                ጐሥዓ፡ ልብ<cb n="b"/><hi rend="rubric">የ፡ ቃለ፡ ሠናየ፡ ጐሥዓ፡ ልብየ፡</hi> ቃለ፡ ሠናየ፡ ወአነ፡ አየድዕ፡
+                                ቅዳ<hi rend="rubric">ሴሃ፡ ለማርያም<supplied reason="omitted">፡</supplied> አኮ፡
+                                    በአብዝኆ፡</hi> አላ፡ በአውህዶ<supplied reason="omitted">፡</supplied> ወአነ፡ አየድዕ<supplied reason="omitted">፡</supplied> ውዳሴሃ፡ ለድንግል።
+                            </incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus to="107ra"/>
```

Comment:
`<locus to="107ra"/>`? Text 3.1 is said to begin on fol. 105rb.

Reply:
The problem is that the beginning of the text was not photographed. 3.1. ends on 105rb, but we don't have its beginning, so I added only the value for the text's ending. I added a comment in the record to make it clearer.